CHANGES.txt

```diff
@@ -1,3 +1,10 @@
+v0.4.1, 03 December 2018
+ * Added direct python API for borrow, e.g. from pmix.borrow import borrow.
+ * Fixed a bug where an Excel workbook would not be created if the path
+ provided was a directory rather than a file. Rather than throwing an error, no
+ error was thrown, and in some cases (e.g. 'borrow'), a success message was
+ still printed.
```

Comment:
I prefer not to wrap CHANGES.txt onto the next line. If you do, then please indent the same amount as the first line.

Reply:
Sure thing. I can change this.

Reply:
(2) Bug when passing directory as outpath
_James wrote:_
> You wrote in CHANGES.txt that "Fixed a bug where an Excel workbook would not be created if the path provided was a directory rather than a file. Rather than throwing an error, no error was thrown, and in some cases (e.g. 'borrow'), a success message was still printed." Can you give a reproducible example?

It may be that you actually ended up fixing this in a recent version. This was broken in 0.2.5, which is what _borrow-web_ was using before I started to fiddle around. I accidentally didn't have your most recent code in the `develop` branch checked out. That was my bad. I'll look further into this and see if the changes I made are actually for preventing any edge case problems at all.

Reply:
Pretty sure that yeah, the problem was just that we were using `0.2.5` in our web apps.